pyboy/plugins/game_wrapper_metroid_ii.pxd

```diff
@@ -0,0 +1,40 @@
+#
+# License: See LICENSE.md file
+# GitHub: https://github.com/Baekalfen/PyBoy
+#
+cimport cython
+from libc.stdint cimport uint8_t
+
+from pyboy.logging.logging cimport Logger
+from pyboy.plugins.base_plugin cimport PyBoyGameWrapper
+
+
+cdef Logger logger
+
+cdef int ROWS, COLS
+
+
+cdef class GameWrapperMetroidII(PyBoyGameWrapper):
+    cdef readonly int _game_over
```

Comment:
`readonly` makes it available on the API. No need to have this, when we have the `game_over` method as well.